Harsh error handling on nonexistent task id

The error handling could stand to be a bit more graceful than a full panic when you give an invalid task id.

Demonstrated below:
```shell
~ ❯❯❯ todoist show 2                                                                                                                                             ⏎
panic: interface conversion: todoist.IDCarrier is nil, not todoist.Item

goroutine 1 [running]:
main.Show(0xc420087340, 0x0, 0x0)
	/home/ocelotsloth/.cache/pacaur/todoist/src/todoist-0.9.1/show.go:20 +0xefd
github.com/urfave/cli.HandleAction(0x7ea120, 0x887be0, 0xc420087340, 0xc42006c600, 0x0)
	/home/ocelotsloth/.cache/pacaur/todoist/src/go/src/github.com/urfave/cli/app.go:490 +0xd4
github.com/urfave/cli.Command.Run(0x86cfbc, 0x4, 0x0, 0x0, 0x0, 0x0, 0x0, 0x874af4, 0x10, 0x0, ...)
	/home/ocelotsloth/.cache/pacaur/todoist/src/go/src/github.com/urfave/cli/command.go:210 +0xb6e
github.com/urfave/cli.(*App).Run(0xc4200de680, 0xc4200101b0, 0x3, 0x3, 0x0, 0x0)
	/home/ocelotsloth/.cache/pacaur/todoist/src/go/src/github.com/urfave/cli/app.go:255 +0x76f
main.main()
	/home/ocelotsloth/.cache/pacaur/todoist/src/todoist-0.9.1/main.go:226 +0x11cf
~ ❯❯❯ todoist show 2274599906                                                                                                                                    ⏎
ID       2274599906
Content  Make breakfast
Project  #Inbox
Labels   
Priority p4
DueDate  17/07/29(Sat)
URL      
~ ❯❯❯

--- a/README.md
+++ b/README.md
@@ -31,7 +31,7 @@ USAGE:
    todoist [global options] command [command options] [arguments...]
 
 VERSION:
-   0.9.1
+   0.9.2
 
 COMMANDS:
      list, l         Shows all tasks

--- a/main.go
+++ b/main.go
@@ -19,7 +19,8 @@ import (
 var (
 	configPath         = os.Getenv("HOME")
 	default_cache_path = os.Getenv("HOME") + "/.todoist.cache.json"
-	CommandFailed      = errors.New("Command Failed")
+	CommandFailed      = errors.New("command failed")
+	IdNotFound         = errors.New("specified id not found")
 	writer             Writer
 )
 
@@ -223,5 +224,8 @@ func main() {
 			Action:  Sync,
 		},
 	}
-	app.Run(os.Args)
+	if err := app.Run(os.Args); err != nil {
+		fmt.Fprintln(os.Stderr, "Error:", err)
+		os.Exit(1)
+	}
 }

--- a/main.go
+++ b/main.go
@@ -40,7 +40,7 @@ func main() {
 	app := cli.NewApp()
 	app.Name = "todoist"
 	app.Usage = "Todoist CLI Client"
-	app.Version = "0.9.1"
+	app.Version = "0.9.2"
 
 	contentFlag := cli.StringFlag{
 		Name:  "content, c",

--- a/modify.go
+++ b/modify.go
@@ -19,11 +19,14 @@ func Modify(c *cli.Context) error {
 
 	var err error
 	item_id, err := strconv.Atoi(c.Args().First())
-	idCarrier, err := todoist.SearchByID(client.Store.Items, item_id)
-	item := idCarrier.(todoist.Item)
 	if err != nil {
 		return err
 	}
+	idCarrier, err := todoist.SearchByID(client.Store.Items, item_id)
+	if err != nil {
+		return IdNotFound
+	}
+	item := idCarrier.(todoist.Item)
 	item.Content = c.String("content")
 	item.Priority = c.Int("priority")
 	item.LabelIDs = func(str string) []int {

--- a/show.go
+++ b/show.go
@@ -13,14 +13,14 @@ func Show(c *cli.Context) error {
 
 	item_id, err := strconv.Atoi(c.Args().First())
 	if err != nil {
-		return err
+		return CommandFailed
 	}
 
 	idCarrier, err := todoist.SearchByID(client.Store.Items, item_id)
-	item := idCarrier.(todoist.Item)
 	if err != nil {
-		return err
+		return IdNotFound
 	}
+	item := idCarrier.(todoist.Item)
 
 	colorList := ColorList()
 	var projectIds []int

